src/nbl/video/device_capabilities/device_features.json

```diff
@@ -203,7 +203,7 @@
                 {
                     "type": "bool",
                     "name": "vertexPipelineStoresAndAtomics",
-                    "value": false,
+                    "value": true,
                     "expose": "MOVE_TO_LIMIT",
```

Comment:
lets not mandate it yet, the androids may not have that feature for a looong time

Reply:
also value needs to be `false`